sim/target-agnostic.mk

```diff
@@ -238,7 +238,7 @@ $(repo_state): $(simulator_verilog) $(fpga_work_dir)/stamp
 	$(firesim_base_dir)/../scripts/repo_state_summary.sh > $(repo_state)
 
 $(fpga_work_dir)/design/$(BASE_FILE_NAME)%: $(simulator_verilog) $(fpga_work_dir)/stamp
-	cp -f $(GENERATED_DIR)/*.ipgen.tcl $(@D)
+	cp -f $(GENERATED_DIR)/*.ipgen.tcl $(@D) || :
```

Comment:
What's ` || :` doing here and why do we need it?  I'm assuming that it is the equivalent to doing ` || true` to ignore non-zero exit code coming from the cp command.  What about your PR introduces the need to do this?

Reply:
This fixes https://github.com/firesim/firesim/pull/918. That PR would never pass builds because the AWS `replace-rtl` would fail since there was no `*.ipgen.tcl` files generated These `ipgen` files are specific to Vitis.

Reply:
I'd prefer this say true if indeed they are equivalent.